pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "tor_archivist"
+version = "0.4.1"
+description = "The officially licensed archivist for /r/TranscribersOfReddit"
+authors = ["Grafeas Group <devs@grafeas.org>"]
+license = "MIT"
+classifiers = [
+    'Development Status :: 1 - Planning',
+    'Intended Audience :: End Users/Desktop',
+    'Topic :: Communications :: BBS'
+]
+repository = "https://github.com/GrafeasGroup/tor_archivist"
+homepage = "https://github.com/GrafeasGroup/tor_archivist"
+exclude = ['test', 'test.*', '*.test.*', '*.test']
+
+[tool.poetry.dependencies]
+python = "^3.7"
```

Comment:
Same as presented before, we're stuck on 3.6 for a little while longer. Unless there have been some code changes that require using 3.7 (in which case, we should put this PR on hold), we should bump this down by a little bit.